backend/src/api/game/game_response.rs

```diff
@@ -0,0 +1,86 @@
+use hive_lib::{
+    bug::Bug, game_status::GameStatus, game_type::GameType, piece::Piece, position::Position,
+};
+use serde::Serialize;
+use serde_with::serde_as;
+use std::collections::HashMap;
+
+#[serde_as]
+#[serde_with::skip_serializing_none]
+#[derive(Serialize)]
+pub struct GameResponse {
+    game_id: u64,
+    turn: usize,
+    game_status: GameStatus,
+    game_type: GameType,
+    tournament: bool,
+    // this should just serialize users later on
+    white_uid: String,
+    white_username: String,
+    white_elo: f64,
+
+    // this should just serialize users later on
+    black_uid: String,
+    black_username: String,
+    black_elo: f64,
+
+    #[serde_as(as = "Vec<((_, _), _)>")]
+    moves: HashMap<(String, Position), Vec<Position>>,
```

Comment:
hmm, shouldn't `HashMap<Piece, Vec<Position>>` be sufficient, since pieces are uniquely identified by their `order`?